distribution/docs/src/main/resources/content/_configuring/map-layers-intrigue-catalog-ui.adoc

```diff
@@ -0,0 +1,42 @@
+:title: Configuring Map for ${catalog-ui} Using Catalog UI Search
```

Comment:
this file name _might_ be too long for Windows. This section also covers more than map layers now. Maybe we could call it `map-configuration.adoc` ?